public/locales/en/translation.json

```diff
@@ -227,17 +227,17 @@
     "questions": {
       "ui-full-time": {
         "tooSick": "Were you too sick or injured to work?",
-        "help-tooSick": "You must be well enough to work everyday of the week to receive full benefits. Check ‘yes’ if a workday was missed due to being sick or injured.",
-        "tooSickNumberOfDays": "If Yes, enter the number of days (1 - 7) you were unable to work.",
+        "help-tooSick": "You must be well enough to work everyday of the week to receive full benefits. Check <strong>Yes</strong> if a workday was missed due to being sick or injured.",
+        "tooSickNumberOfDays": "If yes, enter the number of days (1 - 7) you were unable to work.",
```

Comment:
The big thing here is to not have spaces between the hyphens of the dashes and numbers ....

Reply:
Ah, I thought it was just the capitalization! Ok, that's updated!